### PR TITLE
EOS-23535: Delete unused support dir from utils

### PR DESCRIPTION
**Pull request by vastradparayya**
# Problem Statement
- Remove unused support directory from utils

# Design
-  https://jts.seagate.com/browse/EOS-23535

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: NA
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: NA
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: NA
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N] NA
- [x] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

**Review by sachinpunadikar — APPROVED** [review]
+1

File: py-utils/src/utils/support/0-support_bundle.conf [hunk 1]
```diff
@@ -1,49 +0,0 @@
-# CORTX-Py-Utils: CORTX Python common library.
-# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as published
-# by the Free Software Foundation, either version 3 of the License, or
```
Comment (rahul-tripathi-git) [inline py-utils/src/utils/support/0-support_bundle.conf:5]: Kindly check if we need to move this file to support_framework directory.

**Review by rahul-tripathi-git — CHANGES_REQUESTED** [review]

File: py-utils/src/utils/support/0-support_bundle.conf (hunk 1 above)
Comment (vastradparayya) [reply py-utils/src/utils/support/0-support_bundle.conf:5]: Yes it is present in support_framework directory. Initially we duplicated support directory and renamed that to support_framework.

File: py-utils/setup.py [hunk 2]
```diff
@@ -123,7 +123,7 @@ def get_requirements_files() -> list:
                                  'cortx.conf.sample', 'VERSION',
                                  'src/support/support.yaml',
```
Comment (rahul-tripathi-git) [inline py-utils/setup.py:124]: Please check if this entry needs to be removed.

Comment (vastradparayya) [reply py-utils/setup.py:124]: This config will be copied to rsyslog.d file during build of rpm, previously it was copying from support directory and it is from support_framework

**Review by rahul-tripathi-git — APPROVED** [review]
Looks good to me.

**Pull request merged**